src/test/java/com/exasol/rls/administration/scripts/AddRlsRoleIT.java

```diff
@@ -1,92 +1,83 @@
 package com.exasol.rls.administration.scripts;
 
-import static com.exasol.matcher.ResultSetMatcher.matchesResultSet;
-import static com.exasol.tools.TestsConstants.*;
-import static org.hamcrest.CoreMatchers.containsString;
+import static com.exasol.adapter.dialects.rls.RowLevelSecurityDialectConstants.EXA_ROLES_MAPPING_TABLE_NAME;
+import static com.exasol.matcher.ResultSetStructureMatcher.table;
+import static com.exasol.tools.TestsConstants.PATH_TO_ADD_RLS_ROLE;
+import static com.exasol.tools.TestsConstants.PATH_TO_EXA_RLS_BASE;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import java.io.IOException;
-import java.sql.*;
+import java.sql.Connection;
+import java.sql.SQLException;
 
-import com.exasol.tools.SqlTestSetupManager;
 import org.junit.jupiter.api.*;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
+import org.testcontainers.containers.JdbcDatabaseContainer.NoDriverFoundException;
 import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
 
 import com.exasol.containers.ExasolContainer;
-import com.exasol.containers.ExasolContainerConstants;
 
 // [itest->dsn~add-a-new-role~1]
 @Tag("integration")
 @Testcontainers
-public class AddRlsRoleIT {
-    private static final String EXA_ROLES_MAPPING = "EXA_ROLES_MAPPING";
-    private static final String EXA_ROLES_MAPPING_PROJECTION = "EXA_ROLES_MAPPING_PROJECTION";
+class AddRlsRoleIT extends AbstractAdminScriptIT {
     @Container
-    private static final ExasolContainer<? extends ExasolContainer<?>> container = new ExasolContainer<>(
-            ExasolContainerConstants.EXASOL_DOCKER_IMAGE_REFERENCE);
-    private static Statement statement;
-    private static SqlTestSetupManager sqlTestSetupManager;
+    static final ExasolContainer<? extends ExasolContainer<?>> container = new ExasolContainer<>();
```

Comment:
I think it's more clear to write down with which version of Exasol container we are testing. Even if it's somehow hardcoded in Exasol-test-containers, we can't see it from here. I think we also add test information somewhere to the documentation.

Reply:
I understand what you mean. Let's change that over all our repos in a coordinated effort though.